homeassistant/components/knx/icons.json

```
@@ -0,0 +1,31 @@
+{
+  "entity": {
+    "sensor": {
+      "individual_address": {
+        "default": "mdi:router-network"
+      },
+      "telegrams_incoming": {
+        "default": "mdi:upload-network"
+      },
+      "telegrams_incoming_error": {
+        "default": "mdi:help-network"
+      },
+      "telegrams_outgoing": {
+        "default": "mdi:download-network"
+      },
+      "telegrams_outgoing_error": {
+        "default": "mdi:close-network"
+      },
+      "telegram_count": {
+        "default": "mdi:plus-network"
+      }
+    }
+  },
+  "services": {
+    "send": "mdi:upload-network",
```

In KNX a sent frame is called "telegram". I think the mail-envelope icons would be a good fit. 😃
```suggestion
    "send": "mdi:email-arrow-right",
```